instant.ts

```diff
@@ -26,11 +26,17 @@ interface PackagesMap {
   [packageID: string]: PackageInfo
 }
 
+type EnvironmentVars = Array<{
+  'Environment Variable': string
+  'Default Value': string
+  'Updated Value': string | undefined
```

Comment:
Is this `| undefined` meant to be a default value for the Updated Value? Because I believe it should default to undefined even without this being there? [https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/undefined](undefined)

Reply:
yeah undefined is the type of the default value, so 'Updated Value' is either a string or undefined just for type checks